dist/templates/rbac-ovnkube-master.yaml.j2

```diff
@@ -156,3 +156,35 @@ rules:
     - apiGroups: [""]
       resources: ["configmaps"]
       verbs: ["create", "patch", "update"]
+
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: disable-udn-label-change
+spec:
+  matchConstraints:
+    resourceRules:
+      - apiGroups:   [""]
+        apiVersions: ["v1"]
+        operations:  ["UPDATE"]
+        resources:   ["namespaces"]
+  failurePolicy: Fail
+  validations:
+    - expression: "('k8s.ovn.org/primary-user-defined-network' in oldObject.metadata.labels) == ('k8s.ovn.org/primary-user-defined-network' in object.metadata.labels)"
+      message: "The 'k8s.ovn.org/primary-user-defined-network' label cannot be added/removed after the namespace was created"
```

Comment:
nit:
message: "The 'k8s.ovn.org/primary-user-defined-network' label cannot be added/updated/removed after the namespace was created"